### ISSUE TITLE
Daemonkin Chaos Marines

### ISSUE THREAD

**Issue by lacyj3** [BSData/wh40k-7th-edition#840]
IPhone

Rules error about taking boltguns that can't be resolved no matter what you select.

**Comment by NebSeele** [comment]
Because Equip Boltgun is not appearing in even the Desktop version. I have absolutely no explanation for this, as 
1. this is how the Codex: Chaos Space Marines version of the entry works (the Boltgun is default, you add and subtract from the boltguns and the boltgun min/max changes based on the other two options), 
2.  the entry itself is not marked as hidden and finally
3.  I added a show modifier without ANY conditions to the Equip Boltgun entry while fixing up a Spawn and Bloodcrusher bug.

I'll place this as a bug, Profile Error and a Battlescribe bug for now. I'm looking at other options to streamline the entry so I may have to rework it anyways.

Edit: clarification ammended

**Comment by lacyj3** [comment]
Can you copy it from Chaos Space Marine entry?   

Copy it as it is in the other codex then modify it to new Daemonkin.

**Comment by NebSeele** [comment]
You actually caught me writing an addendum that covers this so I'll write it here:

It would crash my Battlescribe to do it. That's actually the core reason why I chose to override the Black Legion and Crimson Slaughter catalogues with the CSM one once I got the artifacts and rules added to the CSM one (and adjusted the rules accordingly). 

Modifier glitch outs have a horrendous effect on the catalogue editor, prevents you from saving to even correct the problem and has to be done Manually, one by one in order to get them fixed or removed (assuming the program didn't glitch out THAT badly). It was easier and less time consuming to re-write the entries than to copy/paste the CSM ones (I've only done Battle Formations and bug fixes for Daemons, not entry alterations on the levels that CSM needed to appear like the current codices catalogues, so I have no idea what doing one thing in that codex would actually turn out like).

There are way too many modifiers in the CSM entry in both catalogues as is. The Codex CSM one also refers to stuff not in this codex, meaning a massive amount of cleanup before I can even save to check it all worked correctly.

It was easier to scratch write, and therefore in an early enough stage to fix the entry before it becomes bloated. It also serves as a probable way to try and streamline the parent codices, if not the template for their next iterations.
